unattended-upgrade

```diff
@@ -157,7 +157,7 @@ class UnattendedUpgradesCache(apt.Cache):
         """
         try:
             new_cand = ver_in_allowed_origin(pkg, self.allowed_origins)
-            if new_cand != pkg.candidate:
+            if new_cand.version < pkg.candidate.version:
```

Comment:
A comment here why "<" vs "!=" would be good I think. I understand this helps with the case when people whitelist experimental if thats true a comment would be good still.